templates/repo/issue/view_content/pull.tmpl

```diff
@@ -13,7 +13,7 @@
 									{{avatar $.Context .User}}
 								</a>
 							{{end}}
-							<span>
+							<span class="reviewer-info">
```

Comment:
It could `gt-ml-?`

Reply:
Agree, helper classes are better than creating extra classes for simple rules.

Reply:
Yes, it could be `gt-ml-2` and I have fixed it. Thanks!